### PR TITLE
feat: support updating the volta-pinned npm version

### LINKED ISSUE #11011: Update the pinned npm version for Volta
**What would you like Renovate to be able to do?**

Currently, renovate will update the pinned versions of [yarn and node](https://github.com/renovatebot/renovate/blob/main/lib/manager/npm/readme.md) in the volta key. Is it possible to also support updating the pinned npm version?

**Did you already have any implementation ideas?**

I assume it would be similar to the yarn check [here](https://github.com/renovatebot/renovate/blob/e51c8bcdd9abb5deb45b2ffed2340cbdda02996f/lib/manager/npm/extract/index.ts#L213), but I'm not sure if there are other differences.

### PR DESCRIPTION
**Pull request by cjpearson**
## Changes:

Renovate will now update the npm version in the volta section of package.json. Fixes #11011. 

## Context:

Volta is already supported for pinning node and yarn versions. With this change, npm will be supported as well.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [x] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/manager/npm/extract/index.ts [hunk 1]
```diff
@@ -213,6 +213,9 @@ export async function extractPackageFile(
       } else if (depName === 'yarn') {
         dep.datasource = npmId;
         dep.commitMessageTopic = 'Yarn';
+      } else if (depName === 'npm') {
+        dep.datasource = npmId;
+        dep.commitMessageTopic = 'npm';
       } else {
         dep.skipReason = SkipReason.UnknownVolta;
```
Comment (viceice) [inline lib/manager/npm/extract/index.ts:220]: You'll need a new test to get this code covered. 🤗

**Review by viceice — COMMENTED** [review]
I think this needs a new test to get full coverage.

File: lib/manager/npm/extract/index.ts [hunk 2]
```diff
@@ -213,6 +213,9 @@ export async function extractPackageFile(
       } else if (depName === 'yarn') {
         dep.datasource = npmId;
         dep.commitMessageTopic = 'Yarn';
+      } else if (depName === 'npm') {
+        dep.datasource = npmId;
+        dep.commitMessageTopic = 'npm';
```
Comment (rarkins) [inline lib/manager/npm/extract/index.ts:218]: Not needed

**Review by rarkins — CHANGES_REQUESTED** [review]

File: lib/manager/npm/extract/index.ts (hunk 1 above)
Comment (cjpearson) [reply lib/manager/npm/extract/index.ts:220]: I just updated the existing one to also cover the unknown case. Does that work?

File: lib/manager/npm/extract/index.ts (hunk 2 above)
Comment (cjpearson) [reply lib/manager/npm/extract/index.ts:218]: I'm sorry, I'm not sure I understand. Could you explain what's unnecessary?

Comment (rarkins) [inline lib/manager/npm/extract/index.ts:218]:
```suggestion
```

There is no need to specify the `commitMessageTopic` if it's the same as `depName`

Comment (rarkins) [reply lib/manager/npm/extract/index.ts:218]: See new suggestion

Comment (cjpearson) [reply lib/manager/npm/extract/index.ts]: Ah, gotcha. Thanks!

**Review by rarkins — DISMISSED** [review]

**Comment by cjpearson** [comment]
The test failure seems to be unrelated. They pass when I run them locally.

**Review by rarkins — APPROVED** [review]

**Review by viceice — APPROVED** [review]

**Pull request merged**